docs/labwc-actions.5.scd

```diff
@@ -117,6 +117,16 @@ Actions are used in menus and keyboard/mouse bindings.
 	upper-left corner of the window associated with the action. Default is
 	yes.
 
+*<action name="SetDecorations" decorations="value" force_ssd="no" />*
+	Set decorations of focused window.
+
+	*decorations* [full|border|none] On border, only the titlebar will be
+	hidden, borders and resize area are kept. Default is full.
```

Comment:
This implies that the SSD is currently shown, suggest rewording it to something like

```
- *full* enables the whole server side decorations
- *border* only the border and invisible resize area are enabled
- *none* disables everything
```
It doesn't have to be a list.